chainer/_runtime_info.py

```diff
@@ -51,13 +51,32 @@ def __str__(self):
         return s.getvalue()
 
 
-def get_runtime_info():
+def _get_runtime_info():
     return _RuntimeInfo()
 
 
 def print_runtime_info(out=None):
+    """Show Chainer runtime information.
+
+    Runtime information includes:
+    - OS platform
+    - Chainer version
+    - ChainerX version
+    - NumPy version
+    - CuPy version
+      - CUDA information
+      - cuDNN information
+      - NCCL information
+    - iDeep version
+
+    Args:
+        out: Output destination.
+            If it is None, runtime information will be shown in `sys.stdout`.
+
+    """
+    print()
```

Comment:
A drop by but this `print()` might be unintentional?

Reply:
Thanks for catching, fixed in f50b2fb.